### PR TITLE
revision fn

### PR DESCRIPTION
**Pull request by cwehmeyer**
This PR contains most of @franknoe's change requests (see #136). The HMM part is still missing, though.

### CODE REVIEW

File: manuscript/manuscript.tex [hunk 1]
```diff
@@ -112,55 +147,33 @@ \subsection{Essential theory}
...
 
+The subsequent eigenvalues $\lambda_{i>1}(\tau)$ are real with absolute values less than~$1$ and are related to the \emph{characteristic} or \emph{implied} timescales of dynamical processes within the system (eq.~\ref{eq:its}).
+The dynamical process themself (for $i>1$) are encoded by the right eigenvectors $\bm{\psi}_i$,
 \begin{equation}
-\phi_i = \pi \circ \psi_i,
-\label{eq:left-right-eigenvalue-relation}
+\mathbf{P}(\tau)\bm{\psi}_i = \lambda_i(\tau) \bm{\psi}_i,
 \end{equation}
-\noindent{}where $\pi$ is the \emph{stationary distribution} of the MSM, and $\circ$ corresponds to an element-wise vector product.
+where the eigenvalue-eigenvector pairs are indexed in decreasing order.
```
Comment (brookehus) [inline manuscript/manuscript.tex]: in decreasing order of the eigenvalues? do you think this is clear?

File: manuscript/manuscript.tex [hunk 2]
```diff
@@ -112,55 +147,33 @@ \subsection{Essential theory}
...
+The coefficients of the eigenvectors represent the flux into and out of the Markov states that characterize the corresponding process.
+The right eigenvector $\bm{\psi}_1$ is a vector consisting of~$1$'s.
```
Comment (brookehus) [inline manuscript/manuscript.tex]: Should we say something about its interpretation?

File: manuscript/manuscript.tex [hunk 3]
```diff
@@ -175,27 +188,48 @@ \subsection{MSM construction the variational approach}
...
-
-\subsection{Background knowledge and resources}
-\label{sec:background}
-
-For those seeking further resources, ``\emph{Markov State Models: From an Art to a Science}''~\cite{msm-brooke} provides a recent overview, while ``\emph{Markov models of molecular kinetics: Generation and validation}''~\cite{msm-jhp} describes the basic MSM theory and methodology in detail.
-Additionally, two textbooks exist that focus on computational methods and applications~\cite{msm-book} and mathematical theory~\cite{schuette-sarich-book}.
+However, the MSM lag time cannot be optimized using VAMP,
+and must be chosen using a separate validation as described above~\cite{husic2017note}.
+
+Our recommended method for dimensionality reduction, TICA, is a particular implementation of the VAC.
```
Comment (brookehus) [inline manuscript/manuscript.tex]: cite Guillermo's TICA paper

File: manuscript/manuscript.tex [hunk 4]
```diff
@@ -175,27 +188,48 @@ \subsection{MSM construction the variational approach}
...
 
-In addition to publications on theory and application of Markov state modeling~\cite{schuette-msm,buchete-msm-2008,noe-tmat-sampling,bowman-msm-2009,noe-folding-pathways,sarich-msm-quality,noe-fingerprints,noe-dy-neut-scatt,Chodera2014,ben-rev-msm,simon-mech-mod-nmr,oom-feliks,simon-amm},
-we also recommend the literature on TICA~\cite{tica,tica3,tica2,kinetic-maps}, transition path theory (TPT)~\cite{weinan-tpt,metzner-msm-tpt},
-hidden Markov state models (HMMs)~\cite{noe-proj-hid-msm,hmm-baum-welch-alg,hmm-tutorial}, and variational techniques~\cite{noe-vac,vamp-preprint,gmrq}, as these topics play important roles within the standard MSM workflow.
+Dimensionality reduction is achieved by projecting the (mean free) features $\tilde{\mathbf{x}}(t)$
+onto the leading $d$ independent components $\mathbf{U}=[\mathbf{u}_1 \dots \mathbf{u}_d]$,
+\begin{equation}
+\mathbf{y}(t) = \mathbf{U}_d^\top \tilde{\mathbf{x}}(t),
+\end{equation}
+while retaining the kinetic variance
```
Comment (brookehus) [inline manuscript/manuscript.tex]: Cite Noé Clementi JCTC 2015

File: manuscript/manuscript.tex [hunk 5]
```diff
@@ -229,12 +263,14 @@ \subsection{Software/system requirements}
 Should the user prefer not to use Anaconda, a manual installation via the pip installer is possible.
 Alternatively, one can use the Binder service (\url{https://mybinder.org}) to view and run the tutorials online in any browser.
 
-\section{Content and links}
+\section{PyEMMA tutorials}
 
 This tutorial consists of nine Jupyter notebooks which introduce the basic features of PyEMMA.
-The first notebook (00), which we will summarize in the following, showcases the entire estimation, validation, and analysis workflow for a small example system.
+The first notebook (00), which we will summarize in the following, showcases the entire estimation,
```
Comment (brookehus) [inline manuscript/manuscript.tex]: "in the following **section**" ?

File: manuscript/manuscript.tex [hunk 6]
```diff
@@ -251,26 +287,28 @@ \subsection{The PyEMMA workflow}
 \end{itemize}
 
 For the remainder of this manuscript we will walk through the first notebook (00).
-In notebook~00 we analyze a dataset of the Trp-Leu-Ala-Leu-Leu pentapeptide (Fig.~\ref{fig:io-to-tica}a), consisting of~$25$ independent MD trajectories conducted in implicit solvent with frames saved at an interval of~$0.1$~ns.
-We present the results obtained in this notebook, thereby providing an example of how results generated using PyEMMA can be integrated into research publications.
+In notebook~00 we analyze a dataset of the Trp-Leu-Ala-Leu-Leu pentapeptide (Fig.~\ref{fig:io-to-tica}a),
+consisting of~$25$ independent MD trajectories conducted in implicit solvent with frames saved at an interval of~$0.1$~ns.
+We present the results obtained in this notebook,
+thereby providing an example of how results generated using PyEMMA can be integrated into research publications.
 The figures that will be displayed in the following are created in the showcase notebook (00) and can be easily reproduced.
```
Comment (brookehus) [inline manuscript/manuscript.tex]: "in the following **section**"

File: manuscript/manuscript.tex [hunk 7]
```diff
@@ -251,26 +287,28 @@ \subsection{The PyEMMA workflow}
...
 
 Note that the modeler has to select hyper-parameters at most stages throughout the workflow.
 This selection must be done carefully as poor choices make it hard, or even impossible, to build a good MSM.
 
-While there exist automated schemes~\cite{husic-optimized} for cross-validated optimization in the full hyper-parameter 
-space, we chose to adopt a sequential approach where only the hyper-parameters of the current stage are optimized. This 
-approach is not only computationally cheaper but allows us to discuss the significance of the necessary modeling 
-choices.
+While there exist automated schemes~\cite{husic-optimized} for cross-validated optimization in the full hyper-parameter space,
+we chose to adopt a sequential approach where only the hyper-parameters of the current stage are optimized.
```
Comment (brookehus) [inline manuscript/manuscript.tex]: "choose"

We should check the manuscript and make sure we always use "hyper-parameters" and not "hyperparameters"

File: manuscript/manuscript.tex [hunk 8]
```diff
@@ -251,26 +287,28 @@ \subsection{The PyEMMA workflow}
...
+This approach is not only computationally cheaper but allows us to discuss the significance of the necessary modeling choices.
```
Comment (brookehus) [inline manuscript/manuscript.tex]: "but **also** allows us"

File: manuscript/manuscript.tex [hunk 9]
```diff
@@ -251,26 +287,28 @@ \subsection{The PyEMMA workflow}
...
 \begin{figure}
 \includegraphics{figure_2}
-\caption{Example analysis of the conformational dynamics of a pentapeptide backbone: (a)~The Trp-Leu-Ala-Leu-Leu pentapeptide in licorice representation~\cite{vmd}.
+\caption{Example analysis of the conformational dynamics of a pentapeptide backbone:
+(a)~The Trp-Leu-Ala-Leu-Leu pentapeptide in licorice representation~\cite{vmd}.
 (b)~The VAMP-2 score indicates which of the tested featurizations contains the highest kinetic variance.
-(c)~The sample density projected onto the first two time-lagged independent components (ICs) at lag time $\tau=0.5$~ns shows multiple density maxima and
-(d)~the time series of the first two ICs of the first trajectory.}
+(c)~The sample free energy projected onto the first two time-lagged independent components (ICs) at lag time $\tau=0.5$~ns shows multiple minima and
+(d)~the time series of the first two ICs of the first trajectory show rare jumps.}
```
Comment (brookehus) [inline manuscript/manuscript.tex]: "transitions" instead of "jumps" might be more clear

File: manuscript/manuscript.tex [hunk 10]
```diff
@@ -280,20 +318,27 @@ \subsection{Feature selection}
...
 The solid lines in (a) refer to the maximum likelihood result while the dashed lines show the ensemble mean computed with a Bayesian sampling procedure~\cite{ben-rev-msm}.
-The black line indicates where implied timescales are equal to the lag time, whereas the grey area indicates all implied timescales faster than the lag time.
+The black line indicates where implied timescales are equal to the lag time,
+whereas the grey area indicates all implied timescales faster than the lag time.
 In both panels, the (non-grey) shaded areas indicate~$95\%$ confidence intervals computed with the aforementioned Bayesian sampling procedure.}
 \label{fig:its-and-ck}
 \end{figure}
 
-In Markov state modeling our objective is to model the slow dynamics of a molecular process. In order to approximate the slow dynamics in a statistically efficient manner, a lower dimensional representation of our simulation data is necessary.
+In Markov state modeling, our objective is to model the slow dynamics of a molecular process.
```
Comment (brookehus) [inline manuscript/manuscript.tex]: Could reference Schütte J Comput Phys 1999 if you want here

File: manuscript/manuscript.tex [hunk 11]
```diff
@@ -280,20 +318,27 @@ \subsection{Feature selection}
...
+In order to approximate the slow dynamics in a statistically efficient manner,
+a lower dimensional representation of our simulation data is necessary.
 However, the features (e.g. torsion angles, distances or contacts) which best represent the slow dynamical modes of a given molecular system are unknown a priori~\cite{NoeClementiReview}.
```
Comment (brookehus) [inline manuscript/manuscript.tex]: "e.g.~"

File: manuscript/manuscript.tex [hunk 12]
```diff
@@ -280,20 +318,27 @@ \subsection{Feature selection}
...
-Fortunately, the variational principle of conformational dynamics~\cite{noe-vac,nueske-vamk} and the more general variational approach for Markov processes (VAMP)~\cite{vamp-preprint} provide a systematic means to quantitatively compare multiple representations of the simulation data.
+Fortunately, the variational principle of conformational dynamics~\cite{noe-vac,nueske-vamk}
+and the more general variational approach for Markov processes (VAMP)~\cite{vamp-preprint}
```
Comment (brookehus) [inline manuscript/manuscript.tex]: We've already abbreviated VAC and VAMP in an earlier section - no need to define again. Can reference the section maybe.

File: manuscript/manuscript.tex [hunk 13]
```diff
@@ -280,20 +318,27 @@ \subsection{Feature selection}
...
+provide a systematic means to quantitatively compare multiple representations of the simulation data.
 In particular, we can use a scalar score obtained using VAMP to directly compare the ability of certain features to capture slow dynamical modes in a particular molecular system.
 
 Here, we utilize the VAMP-2 score, which maximizes the kinetic variance contained in the features~\cite{kinetic-maps}.
```
Comment (brookehus) [inline manuscript/manuscript.tex]: I would also cite the VAMP paper in the end of sentence citation.

File: manuscript/manuscript.tex [hunk 14]
```diff
@@ -304,38 +349,42 @@ \subsection{Feature selection}
...
+
+To test the validity of our MSM, we perform a Chapman-Kolmogorov (CK) test.
+Visualizing the full transition probability matrix $T$ is difficult;
+we therefore coarse-grain $T$ into a smaller number of metastable states before performing the test.
 An appropriate number of metastable states can be chosen by identifying a relatively large gap in the ITS plot.
 For this analysis, we chose five metastable states.
-The CK test (Fig.~\ref{fig:its-and-ck}b) shows that predictions from our MSM (blue-dashed lines) agrees well with MSMs estimated with longer lag times (black-solid lines)
-Thus, the CK test confirms that five metastable states is an appropriate choice and shows that the MSM we have estimated at lag time $\tau=0.5$~ns indeed predicts the long-timescale behavior of our system within error (blue/shaded area).
+The CK test (Fig.~\ref{fig:its-and-ck}b) shows that predictions from our MSM (blue-dashed lines)
+agrees well with MSMs estimated with longer lag times (black-solid lines)
```
Comment (brookehus) [inline manuscript/manuscript.tex]: "predictions ... **agree** well"

File: manuscript/manuscript.tex [hunk 15]
```diff
@@ -369,14 +424,18 @@ \subsection{Analyzing the MSM}
 \end{equation}
 where $\pi_j$ denotes the MSM stationary weight of the $j^\textrm{th}$ microstate.
 
-In order to interpret the slowest relaxation timescales, we refer to the (right) eigenvectors, as they are independent of the stationary distribution.
+In order to interpret the slowest relaxation timescales, we refer to the (right) eigenvectors,
+as they are independent of the stationary distribution.
```
Comment (brookehus) [inline manuscript/manuscript.tex]: can reference the theory section here

File: manuscript/manuscript.tex [hunk 16]
```diff
@@ -369,14 +424,18 @@ \subsection{Analyzing the MSM}
...
 This enables us to specifically study what conformational changes are happening on a particular time scale independently of the equilbrium distribution.
```
Comment (brookehus) [inline manuscript/manuscript.tex]: we should ctrl+f for "time scale" vs "timescale"

Comment (brookehus) [reply manuscript/manuscript.tex]: I think we tend towards "timescale"

File: manuscript/manuscript.tex [hunk 17]
```diff
@@ -418,18 +479,25 @@ \subsection{Connecting the MSM with experimental data}
...
-Now that we have an approximation of the fluorescence in each of our MSM states, we can use PyEMMA to compute the fluorescence autocorrelation function (ACF) from our MSM (\ref{fig:msm-exp-obs}a).
+Since we have both the stationary and dynamic properties of the molecular system encoded in the MSM transition probability matrix,
+we can compute observables that involve both stationary ensemble averages as well as correlation functions.
+
+As an example, here we look at the fluorescence correlation of Trp-1,
+since this terminal tryptophan is a realistic experimental observable for our pentapeptide system.
+In order to compute the fluorescence correlation functions we require a microscopic,
+instantaneous value of the tryptophan fluorescence for each of the original~$75$ MSM microstates.
+To approximate the fluorescence signal in our pentapeptide system,
+we use the mdtraj library~\cite{mdtraj} to compute the solvent accessible surface area (SASA) of Trp-1.
```
Comment (brookehus) [inline manuscript/manuscript.tex]: Can also cite Shrake, A; Rupley, JA. (1973) J Mol Biol 79 (2): 351--71 when you write (SASA), for the actual algorithm

File: manuscript/manuscript.tex [hunk 18]
```diff
@@ -438,38 +506,52 @@ \subsection{Summary}
...
+we can integrate unbiased and biased simulations in a systematic manner to speed up the convergence. 
+MEMMs consequently enable users to combine enhanced sampling methods such as umbrella sampling or replica exchange
+with conventional molecular dynamics simulations to more efficiently study rare event kinetics~\cite{trammbar}. 
 MEMMs are implemented in PyEMMA.
 
 Another issue often faced during Markov state modeling is a lack of quantitative agreement with complementary experimental data. 
-This issue is not intrinsic to the Markov state modeling approach as such, but rather associated with systematic errors in the force field model used to conduct the simulation. 
-Nevertheless, using Augmented Markov models (AMM) it is possible to build an integrative MSM which balances experimental and simulation data, taking into account their respective uncertainties~\cite{simon-amm}. 
+This issue is not intrinsic to the Markov state modeling approach as such,
+but rather associated with systematic errors in the force field model used to conduct the simulation.
```
Comment (brookehus) [inline manuscript/manuscript.tex]: "but rather **is** associated"

Comment (brookehus) [reply manuscript/manuscript.tex]: is there a citation for this claim?

File: manuscript/manuscript.tex [hunk 19]
```diff
@@ -438,38 +506,52 @@ \subsection{Summary}
...
-Other deep learning methods for performing the dimension reduction~\cite{tae}, finding reaction coordinates for enhanced sampling~\cite{hernandez-vde,Sultan2018-vde-enhanced-sampling,Ribeiro2018-rave}, and generative MSMs~\cite{deep-gen-msm-preprint} have been put forward and are likely to spawn an active field of research on its own right.
+Recently, there have been steps towards replacing the traditional user-directed pipeline (involving featurizing,
+reducing dimension, discretizing, MSM estimation and coarse-graining) by a single end-to-end deep learning method such as VAMPnets~\cite{vampnet}.
+Other deep learning methods for performing the dimension reduction~\cite{tae},
+finding reaction coordinates for enhanced sampling~\cite{hernandez-vde,Sultan2018-vde-enhanced-sampling,Ribeiro2018-rave},
+and generative MSMs~\cite{deep-gen-msm-preprint} have been put forward and are likely to spawn an active field of research on its own right.
```
Comment (brookehus) [inline manuscript/manuscript.tex]: "**in** its own right"

File: manuscript/manuscript.tex (hunk 18 above)
Comment (cwehmeyer) [reply manuscript/manuscript.tex]: > is there a citation for this claim?

@psolsson?

**Comment by cwehmeyer** [comment]
Thanks a lot for the review, @brookehus!

**Pull request merged**